lib/Doctrine/Common/DataFixtures/Purger/ORMPurger.php

```diff
@@ -81,6 +86,7 @@ public function getPurgeMode()
     public function setEntityManager(EntityManagerInterface $em)
     {
         $this->em = $em;
+        $this->invalidateCache();
```

Comment:
no need to create a private method for that IMO

Reply:
done